omnigibson/examples/robots/import_custom_robot.py

```diff
@@ -0,0 +1,977 @@
+        "radius": 0.06
+      - "center": [-0.34, -0.027, -0.03]
+        "radius": 0.06
+    right_arm_link3:
+      - "center": [0.037, -0.058, -0.044]
+        "radius": 0.05
+      - "center": [0.095, -0.08, -0.044]
+        "radius": 0.03
+      - "center": [0.135, -0.08, -0.043]
+        "radius": 0.03
+      - "center": [0.176, -0.08, -0.043]
+        "radius": 0.03
+      - "center": [0.22, -0.077, -0.043]
+        "radius": 0.03
+    right_arm_link4:
+      - "center": [-0.002, 0.0, 0.276]
+        "radius": 0.04
+    right_arm_link5:
+      - "center": [0.059, -0.001, -0.021]
+        "radius": 0.035
+    right_arm_link6:
+      - "center": [-0.0, 0.0, 0.04]
+        "radius": 0.035
+    wheel_link1:
+      - "center": [-0.0, 0.0, -0.03]
+        "radius": 0.06
+    wheel_link2:
+      - "center": [0.0, 0.0, 0.03]
+        "radius": 0.06
+    wheel_link3:
+      - "center": [0.0, 0.0, -0.03]
+        "radius": 0.06
+    left_gripper_link1:
+      - "center": [-0.03, 0.0, -0.002]
+        "radius": 0.008
+      - "center": [-0.01, 0.0, -0.003]
+        "radius": 0.007
+      - "center": [0.005, 0.0, -0.005]
+        "radius": 0.005
+      - "center": [0.02, 0.0, -0.007]
+        "radius": 0.003
+    left_gripper_link2:
+      - "center": [-0.03, 0.0, -0.002]
+        "radius": 0.008
+      - "center": [-0.01, 0.0, -0.003]
+        "radius": 0.007
+      - "center": [0.005, 0.0, -0.005]
+        "radius": 0.005
+      - "center": [0.02, 0.0, -0.007]
+        "radius": 0.003
+    right_gripper_link1:
+      - "center": [-0.03, 0.0, -0.002]
+        "radius": 0.008
+      - "center": [-0.01, -0.0, -0.003]
+        "radius": 0.007
+      - "center": [0.005, -0.0, -0.005]
+        "radius": 0.005
+      - "center": [0.02, -0.0, -0.007]
+        "radius": 0.003
+    right_gripper_link2:
+      - "center": [-0.03, 0.0, -0.002]
+        "radius": 0.008
+      - "center": [-0.01, 0.0, -0.003]
+        "radius": 0.007
+      - "center": [0.005, 0.0, -0.005]
+        "radius": 0.005
+      - "center": [0.02, 0.0, -0.007]
+        "radius": 0.003
+  default_qpos:                     # (list of float): Default joint configuration
+    - 0.0
+    - 0.0
+    - 0.0
+    - 0.0
+    - 0.0
+    - 0.0
+    - 1.906
+    - 1.906
+    - -0.991
+    - -0.991
+    - 1.571
+    - 1.571
+    - 0.915
+    - 0.915
+    - -1.571
+    - -1.571
+    - 0.03
+    - 0.03
+    - 0.03
+    - 0.03
+
+"""
+
+
+def create_rigid_prim(stage, link_prim_path):
+    """
+    Creates a new rigid link prim nested under @root_prim
+
+    Args:
+        stage (Usd.Stage): Current active omniverse stage
+        link_prim_path (str): Prim path at which link will be created. Should not already exist on the stage
+
+    Returns:
+        Usd.Prim: Newly created rigid prim
+    """
+    # Make sure link prim does NOT already exist (this should be a new link)
+    link_prim_exists = lazy.omni.isaac.core.utils.prims.is_prim_path_valid(link_prim_path)
```

Comment:
There's a `stage` argument but this line looks on the open Isaac Sim stage

Reply:
fixed